source/documentation/deploying-an-app/using-circleci-for-continuous-deployment.html.md.erb

```diff
@@ -0,0 +1,262 @@
+---
+title: Continuous Deployment of an application using CircleCI and Helm
+last_reviewed_on: 2019-12-09
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+## Introduction
+This document covers how to continuously deploy your application in the Cloud Platform. 
+
+> Note: This document is specific to using [CircleCI](https://circleci.com/) as the Continuous Integration method.*
+
+### Objective
+By the end of the tutorial, you will have done the following:
+
+- Created a Service Account for CircleCI in your namespace
+- Generated a CircleCI configuration file in your application repository. The configuration file will authenticate with your chosen cluster, build a docker image and push it to ECR and upgrade your helm deployment with the new docker image.
+- Have an automated CircleCI pipeline that upgrades your helm deployment when a new change is pushed to your master branch
+
+### Requirements
+It is assumed you have the following:
+
+ - You have [created an environment for your application][env-create]
+ - You have deployed the [multi-container rails app] to the 'cloud-platform-live-1' cluster.
```

Comment:
Sorry, I missed this one. We usually refer to the cluster as either "live-1" or "the cloud platform" so the "cloud-platform-live-1" cluster is a bit ambiguous. Personally, I'd use the "live-1" cluster.

Reply:
done